The type uint is not defined by musl libc headers.

--- ksysguard/ksysguardd/Linux/netstat.c	2015-05-07 20:13:26.000000000 +0200
+++ ksysguard/ksysguardd/Linux/netstat.c	2015-10-09 00:02:49.350628001 +0200
@@ -32,6 +32,10 @@
 #include "ccont.h"
 #include "netstat.h"
 
+#if !defined(__GLIBC__)
+typedef unsigned int uint;
+#endif
+
 static CONTAINER TcpSocketList = 0;
 static CONTAINER UdpSocketList = 0;
 static CONTAINER UnixSocketList = 0;
